mitosheet/mitosheet/tests/step_performers/test_export_to_file.py

```diff
@@ -424,13 +471,8 @@ def test_transpiled_with_export_to_xlsx_conditional_and_rows(column_ids, filters
     })
     filename = 'test_format_conditional_and_rows.xlsx'
     mito.export_to_file('excel', [0], filename)
-    font_color_str = f'color: {font_color}; ' if font_color is not None else ''
-    background_color_str = f'background-color: {background_color}' if background_color is not None else ''
-    if background_color is None:
-        font_color_str = font_color_str[:-2]
-    assert "\n".join(mito.transpiled_code) == f"""from mitosheet.public.v3 import *
-import pandas as pd
-import numpy as np
+    assert "\n".join(mito.transpiled_code[:-2]) == f"""from mitosheet.public.v3 import *
+import pandas as pd{f"{chr(10)}import numpy as np" if number_formatting else ''}
```

Comment:
This was a workaround but I actually ended up realizing I could just move that whole expression into a variable definition to avoid using chr - just pushed!